Mathlib/Data/Real/Basic.lean

```diff
@@ -607,3 +607,11 @@ def IsPowMul {R : Type*} [Pow R ℕ] (f : R → ℝ) :=
 def RingHom.IsBoundedWrt {α : Type*} [Ring α] {β : Type*} [Ring β] (nα : α → ℝ) (nβ : β → ℝ)
     (f : α →+* β) : Prop :=
   ∃ C : ℝ, 0 < C ∧ ∀ x : α, nβ (f x) ≤ C * nα x
+
+/-- If `x` is multiplicative with respect to `f`, then so is any `x^n`. -/
+theorem pow_isMul_of_isMul {R : Type*} [MonoidWithZero R] (f : R → ℝ) {x : R}
```

Comment:
I am a bit sceptical about the name of this theorem (and the same applies for all `is_mul` and `isMul` in the file `SmoothingSeminorm`: we do not have (AFAIK) a `IsMul` class, whereas this naming in inspired from that. I would probably go for `pow_mul_apply_eq_pow_mul`, or something like this.

Reply:
Oh, actually there is
https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Group/Center.html#IsMulCentral

Is it worth using that API? At any rate I would then replace all names `IsMul` with `IsMulCentral` to be aligned.

Reply:
I don't see how to use `IsMulCentral` here (there is no map involved in that definition), but I am ok with renaming this lemma to `pow_mul_apply_eq_pow_mul`.